pyvisa/attributes.py

```diff
@@ -389,6 +389,22 @@ def pre_set(self, value: Union[str, bytes]) -> int:
 # --- Session attributes ---------------------------------------------------------------
 # Attributes are in the same order as in the constants.ResourceAttribute enum
 
+
+# class AttrVI_ATTR_RM_SESSION(Attribute):
+#     """Specifies the session of the Resource Manager used to open this session."""
+#
+#     resources = AllSessionTypes
+#
+#     py_name = "resource_manager_session"
+#
+#     visa_name = "VI_ATTR_RM_SESSION"
+#
+#     visa_type = "ViSession"
+#
+#     default = NotAvailable
+#
+#     read, write, local = True, False, False
+#
```

Comment:
Am I wrong in thinking that if you fully remove it you will run into your first issue will re-appear ? I think that we want to allow people that want to use get_visa_attribute to use it but just not provide multiple high-level ways to do the same thing.

So I would un-comment this (and clear the py_name).